src/frontends/paddle/src/op/matmul_v2.cpp

```diff
@@ -14,7 +14,14 @@ NamedOutputs matmul_v2(const NodeContext& node) {
     const auto transpose_a = node.get_attribute<bool>("trans_x", false);
     const auto transpose_b = node.get_attribute<bool>("trans_y", false);
     const auto mm = std::make_shared<default_opset::MatMul>(x, y, transpose_a, transpose_b);
-    return node.default_single_output_mapping({mm}, {"Out"});
+    const auto& mm_rank = mm->get_output_partial_shape(0).rank();
+
+    std::shared_ptr<Node> result = mm;
+    if (mm_rank.is_static() && mm_rank.get_length() == 0) {
```

Comment:
Probably it is more concise to use is_scalar(mm->get_output_partial_shape(0)).

Reply:
updated